### PR TITLE
Lang processor interlayer

### PR DESCRIPTION
**Pull request by SergeyAsm**
added script engine interlayer

### CODE REVIEW

File: wicket-console/pom.xml [hunk 1]
```diff
@@ -35,6 +35,47 @@
 	    <artifactId>wicket-devutils</artifactId>
 	    <scope>provided</scope>
     </dependency>
+    		<dependency>
```
Comment (PhantomYdn) [inline wicket-console/pom.xml]: See previous PR for comments about dependencies.

File: wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/IScriptEngineInterlayer.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package ru.ydn.wicket.wicketconsole;
+
+import javax.script.ScriptException;
+
+public interface IScriptEngineInterlayer {
+	
+	String getName();
+	
+	IScriptEngineInterlayerResult eval(String command) throws Exception;
```
Comment (PhantomYdn) [inline wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/IScriptEngineInterlayer.java]: Is context saving somewhere?

File: wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/ScriptEngineInterlayerManager.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.Map;
+
+public class ScriptEngineInterlayerManager {
+
+	public static final ScriptEngineInterlayerManager INSTANCE = new ScriptEngineInterlayerManager();
+
+	Map<String,IScriptEngineInterlayer> interlayers;
+	
+	private ScriptEngineInterlayerManager() {
+		interlayers = new HashMap<String,IScriptEngineInterlayer>();
```
Comment (PhantomYdn) [inline wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/ScriptEngineInterlayerManager.java]: If I understood this correctly: execution context will be the same for all users who is working in parallel. And that's definetly wrong.

**Review by PhantomYdn — CHANGES_REQUESTED** [review]

File: wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/ScriptEngineInterlayerManager.java (hunk 3 above)
Comment (SergeyAsm) [reply wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/ScriptEngineInterlayerManager.java]: Mea culpa )

File: wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/IScriptEngineInterlayer.java (hunk 2 above)
Comment (SergeyAsm) [reply wicket-console/src/main/java/ru/ydn/wicket/wicketconsole/IScriptEngineInterlayer.java]: just IDE trash

**Review by PhantomYdn — APPROVED** [review]

**Comment by PhantomYdn** [comment]
Merged separatly from this PR

**Pull request closed without merging**